### DISCUSSION TITLE
「开源之史」系列之九/九：推动 GNU/Linux —— 企业联盟式的非营利基金会 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#72] [Announcements]
# 「开源之史」系列之九/九：推动 GNU/Linux —— 企业联盟式的非营利基金会 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/07-01-the-rise-of-linux-foundataion/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

这段历史, 的确无人描述过, 
只知道以往因为 Linux 足够稳定后,
因为其免费特性, 可以极大减少基础设施运营成本,
导致除非 windows only 的系统, 都迁移到 Linux 环境中,
变成 IT 系统长期运营首选系统,
市场才因为这个现实建立起来;

只是, 这个拐点具体在哪个事件之后, 真没注意;

值得继续挖掘, 绘制时间线....
